StubExample/StubExample/ViewController.swift

```diff
@@ -10,17 +10,25 @@ import UIKit
 import Metal
 import MetalPerformanceShadersStub
 
+
 class ViewController: UIViewController {
 
+    @IBOutlet weak var label: UILabel!
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        if let device = MTLCreateSystemDefaultDevice() {
-            let _ = MPSImage.init(device: device, imageDescriptor: MPSImageDescriptor())
-        }
 
-        MPSMatrixDescriptor.rowBytes(fromColumns: 0, dataType: .float32) // Crashes always because it uses just the Stub
-
-        var _: CVMetalTextureCache
```

Comment:
We should keep the `CVMetalTextureCache` somewhere. The idea is to show the user that it works on a simulator (that it can be used as a type, or whatever).